src/main/java/com/stripe/param/SubscriptionCreateParams.java

```diff
@@ -0,0 +1,758 @@
+   * application fee percentage. For more information, see the application fees
+   * [documentation](https://stripe.com/docs/connect/subscriptions#collecting-fees-on-subscriptions).
+   */
+  @SerializedName("application_fee_percent")
+  BigDecimal applicationFeePercent;
+
+  /**
+   * Either `charge_automatically`, or `send_invoice`. When charging automatically, Stripe will
+   * attempt to pay this subscription at the end of the cycle using the default source attached to
+   * the customer. When sending an invoice, Stripe will email your customer an invoice with payment
+   * instructions. Defaults to `charge_automatically`.
+   */
+  @SerializedName("billing")
+  Billing billing;
+
+  /**
+   * A future timestamp to anchor the subscription's [billing
+   * cycle](https://stripe.com/docs/subscriptions/billing-cycle). This is used to determine the date
+   * of the first full invoice, and, for plans with `month` or `year` intervals, the day of the
+   * month for subsequent invoices.
+   */
+  @SerializedName("billing_cycle_anchor")
+  String billingCycleAnchor;
+
+  /**
+   * Define thresholds at which an invoice will be sent, and the subscription advanced to a new
+   * billing period. Pass an empty string to remove previously-defined thresholds.
+   */
+  @SerializedName("billing_thresholds")
+  Object billingThresholds;
+
+  /**
+   * Boolean indicating whether this subscription should cancel at the end of the current period.
+   */
+  @SerializedName("cancel_at_period_end")
+  Boolean cancelAtPeriodEnd;
+
+  /**
+   * The code of the coupon to apply to this subscription. A coupon applied to a subscription will
+   * only affect invoices created for that particular subscription.
+   */
+  @SerializedName("coupon")
+  String coupon;
+
+  /** The identifier of the customer to subscribe. */
+  @SerializedName("customer")
+  String customer;
+
+  /**
+   * Number of days a customer has to pay invoices generated by this subscription. Valid only for
+   * subscriptions where `billing` is set to `send_invoice`.
+   */
+  @SerializedName("days_until_due")
+  Long daysUntilDue;
+
+  /**
+   * ID of the default payment source for the subscription. It must belong to the customer
+   * associated with the subscription and be in a chargeable state. If not set, defaults to the
+   * customer's default source.
+   */
+  @SerializedName("default_source")
+  String defaultSource;
+
+  /** Specifies which fields in the response should be expanded. */
+  @SerializedName("expand")
+  List<String> expand;
+
+  /** List of subscription items, each with an attached plan. */
+  @SerializedName("items")
+  List<Item> items;
+
+  /**
+   * A set of key-value pairs that you can attach to a `Subscription` object. It can be useful for
+   * storing additional information about the subscription in a structured format.
+   */
+  @SerializedName("metadata")
+  Map<String, String> metadata;
+
```

Comment:
We're missing `plan` though I guess it's on purpose because we consider it deprecated and we're fine forcing merchants to not support it? (it sounds okay especially as there's always the escape hatch anyway, just asking to confirm)

Reply:
@remi-stripe I still haven't addressed this "missing" `plan` field and also Customer create too.
Are we okay with releasing without first?